pkg/scheduler/scheduler.go

```diff
@@ -419,6 +419,8 @@ func (sched *Scheduler) finishBinding(prof *profile.Profile, assumed *v1.Pod, ta
 // scheduleOne does the entire scheduling workflow for a single pod.  It is serialized on the scheduling algorithm's host fitting.
 func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	podInfo := sched.NextPod()
+	podPriority := podutil.GetPodPriority(podInfo.Pod)
```

Comment:
this should come after the klog.V(3) statement that announces that we're attempting to schedule this pod